session/handler.go

```diff
@@ -459,6 +485,92 @@ func (h *Handler) IsAuthenticated(wrap httprouter.Handle, onUnauthenticated http
 	}
 }
 
+// swagger:parameters adminSessionRefresh
+// nolint:deadcode,unused
+type adminSessionRefresh struct {
+	// ID is the session's ID.
+	//
+	// required: true
+	// in: path
+	ID string `json:"id"`
+}
+
+// swagger:route PATCH /sessions/refresh/{id} v0alpha2 adminSessionRefresh
+//
+// Calling this endpoint refreshes a given session.
+// If `session.earliest_refresh` is set it will only refresh the session after this time has passed.
```

Comment:
```suggestion
// Calling this endpoint refreshes the given session ID.
// If `session.earliest_refresh` is set it will only refresh the session after this time has passed.
```